README.md

```diff
@@ -18,6 +18,9 @@ See Installation instructions for:
 * ROS2 Galactic:  https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Binary.html
 
 
+1. Switch to the `surface_plugin` branch in ign-gazebo and recompile.
```

Comment:
remove this since it's more for developers? We'll just have to assume that users are going to be installing ign-gazebo from the debs.